sdk/python/kfp/compiler/compiler_test.py

```diff
@@ -524,12 +524,12 @@ def dummy_op(msg: str = ''):
 
         with self.assertRaisesRegex(
                 compiler_utils.InvalidTopologyException,
-                r'Illegal task dependency across DSL context managers\. A downstream task cannot depend on an upstream task within a dsl\.Condition context unless the downstream is within that context too\. Found task dummy-op which depends on upstream task producer-op within an uncommon dsl\.Condition context\.'
+                r'Illegal task dependency across DSL context managers\. A downstream task cannot depend on an upstream task within a dsl\.If context unless the downstream is within that context too\. Found task dummy-op which depends on upstream task producer-op within an uncommon dsl\.If context\.'
         ):
 
             @dsl.pipeline(name='test-pipeline')
             def my_pipeline(val: bool):
-                with dsl.Condition(val == False):
```

Comment:
Given `dsl.Condition` is not deprecated yet, we need to ensure it still functions. From this perspective, we should keep its references in test.

Reply:
Closed this PR.